assignment/week3/lunchMenu/src/pages/RecommendStartPage.jsx

```diff
@@ -0,0 +1,51 @@
+import styled from 'styled-components';
+import RandomPage from './RandomPage';
+import PreferencePage from './PreferencePage';
+import { useState } from 'react';
+const RecommendStartPage = (props) => {
```

Comment:
다른 모든 컴포넌트에도 해당되는 얘기인데, 구조분해할당을 이용해서 props를 더 가독성 좋게 사용해보면 어때요?!
```suggestion
const RecommendStartPage = ({category, startRecommend, setStartRecommend}) => {
```
이렇게!
그런데 만약 prop이 더 많아지면 이렇게 매개변수에 바로 구조분해할당을 하는게 너무 길어져서 가독성이 안좋아질 수도 있어요. 그럴 때는 보통 이렇게 많이 써요!

const {category, startRecommend, setStartRecommend} = props;

이렇게하면 category, startRecommend, setStartRecommend라는 const 변수를 선언함과 동시에 prop으로 받아온 그 값들을 할당해주어서, 이 밑으로는 props.category라고 쓰는게 아니라 바로 category 라고만 써서 사용할 수 있어요.

Reply:
넘 친절한 설명 감사합니다 〰️ ♥